packages/flutter/lib/src/widgets/image.dart

```diff
@@ -459,6 +551,182 @@ class Image extends StatefulWidget {
   /// The image to display.
   final ImageProvider image;
 
+  /// A builder function responsible for creating the widget that represents
+  /// this image.
+  ///
+  /// If this is null, this widget will display an image that is painted as
+  /// soon as the first image frame is available (and will appear to "pop" in
+  /// if it becomes available asynchronously). Callers might use this builder to
+  /// add effects to the image (such as fading the image in when it becomes
+  /// available) or to display a placeholder widget while the image is loading.
+  ///
+  /// To have finer-grained control over the way that an image's loading
+  /// progress is communicated to the user, see [loadingBuilder].
+  ///
+  /// ## Chaining with [loadingBuilder]
+  ///
+  /// If a [loadingBuilder] has _also_ been specified for an image, the two
+  /// builders will be chained together, and the _result_ of this builder will
```

Comment:
```suggestion
  /// builders will be chained together: the _result_ of this builder will
```

The "and" made it sound to me like the builders are chained together fist and then that result is passed to `loadingBuilder`. The colon makes it more obvious to me that the following is how the chaining happens.

Reply:
Done